### PR TITLE
[FLINK-19105][docs]update demo error

### PR DESCRIPTION
**Pull request by YongjinZhou**
## What is the purpose of the change
This pull request correct demo error

## Brief change log
tableEnv.connect(new FileSystem("/path/to/file")
->
tableEnv.connect(new FileSystem().path("/path/to/file"))

DataTypes.LONG()
->DataTypes.BIGINT()

## Verifying this change

This change added tests and can be verified as follows:
before
image
after
image

## Documentation

  - Does this pull request introduce a new feature? (yes / no) no
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by YongjinZhou** [comment]
cc @dianfu  could plz have a review?

File: docs/dev/table/common.md [hunk 1]
```diff
@@ -743,9 +743,9 @@ TableEnvironment tableEnv = ...; // see "Create a TableEnvironment" section
 final Schema schema = new Schema()
     .field("a", DataTypes.INT())
     .field("b", DataTypes.STRING())
-    .field("c", DataTypes.LONG());
+    .field("c", DataTypes.BIGINT());
 
-tableEnv.connect(new FileSystem("/path/to/file"))
+tableEnv.connect(new FileSystem().path("/path/to/file"))
```
Comment (dianfu) [inline docs/dev/table/common.md:748]: This error exists in several places in the documentation. Could you also update them?

Comment (YongjinZhou) [reply docs/dev/table/common.md:748]: > This error exists in several places in the documentation. Could you also update them?

@dianfu ok， It has been updated.

**Review by dianfu — APPROVED** [review]
LGTM.

**Comment by dianfu** [comment]
@YongjinZhou Good work! Merging... 

PS: Could you let me know your JIRA account(you can comment on the JIRA page), then I can assign this ticket to you? :)

**Pull request closed without merging**